bundler/spec/bundler/stub_specification_spec.rb

```diff
@@ -55,4 +55,24 @@
       expect(stub.missing_extensions?).to be true
     end
   end
+
+  describe "#activated?" do
+    it "returns true after activation" do
+      stub = described_class.from_stub(with_bundler_stub_spec)
+
+      expect(stub.activated?).to be false
+      stub.activated = true
+      expect(stub.activated?).to be true
+    end
+
+    it "returns true after activation if the underlying stub is a `Gem::StubSpecification`" do
+      spec_path = File.join(File.dirname(__FILE__), "specifications", "foo.gemspec")
+      gem_stub = Gem::StubSpecification.new(spec_path, File.dirname(__FILE__),"","")
```

Comment:
I'm not quite sure which codepath this ends up exercising but in the end the test would fail with this.

Reply:
Yeah, this seems ok to me. The reason the other test works already is because `Gem::Specification` aliases `#activated?` to `#activated`, but `Gem::StubSpecification` does not do that.